cypress/elements/dashboardItem.js

```diff
@@ -0,0 +1,70 @@
+const clickMenuButton = itemUid => {
+    cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+        .find('[data-test="item-menu-button"]')
+        .click()
+}
+
+const clickViewAsChart = () => {
+    cy.contains('View as Chart').click()
+}
+
+const clickViewAsTable = () => {
+    cy.contains('View as Table').click()
+}
+
+const clickViewAsMap = () => {
+    cy.contains('View as Map').click()
+}
+
+const expectChartToExist = itemUid => {
+    if (itemUid) {
+        cy.get(`[data-test="dashboard-item-prog-${itemUid}"]`)
+            .find('.highcharts-container')
+            .should('exist')
+            .and('be.visible')
+    } else {
+        cy.get('.highcharts-container').should('exist').and('be.visible')
```

Comment:
Shouldn't this part somehow be filtered by the `itemUid`? Or how do you know that you've found the right `.highcharts-container` and not just any?

Reply:
Removed the else - not in use.